.github/workflows/release-embedding-sdk.yml

```diff
@@ -249,7 +254,7 @@ jobs:
         run: |
           echo //registry.npmjs.org/:_authToken=${{ secrets.NPM_RELEASE_TOKEN }} > .npmrc
           # Please keep the value in sync with `inputs.branch`'s release branch
-          npm publish --tag ${{fromJson('{"master": "canary", "release-x.51.x": "51-stable"}')[inputs.branch]}}
+          npm publish --tag ${{fromJson('{"master": "nightly", "release-x.51.x": "51-stable"}')[inputs.branch]}}
```

Comment:
We would [rename `canary` to `nightly`](https://metaboat.slack.com/archives/C063Q3F1HPF/p1731393056310439?thread_ts=1730986265.758069&cid=C063Q3F1HPF). This will match Metabase's tags more, although we don't deploy `master` nightly 😅